{"name": "test_add[positive1]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 20 == 15\n  +20\n  -15", "trace": "self = <testcases.test_cacu_yaml.TestCalc object at 0x000001E4E8C350C8>, setprint = None, fristofall = None, a = 10, b = 10, expect = 15\n\n    @pytest.mark.run(order=1)\n    @pytest.mark.parametrize('a,b,expect',datas['add']['datas'],ids=datas['add']['ids'])\n    def test_add(self,setprint,fristofall, a, b, expect):\n        # calc = Calculator()\n        result = self.calc.add(a, b)\n>       assert result == expect\nE       assert 20 == 15\nE         +20\nE         -15\n\ntest_cacu_yaml.py:25: AssertionError"}, "parameters": [{"name": "a", "value": "10"}, {"name": "b", "value": "10"}, {"name": "expect", "value": "15"}], "start": 1604461033041, "stop": 1604461033041, "uuid": "0611c9fb-36d8-4839-9e69-2ca21537f891", "historyId": "460267c27f951306db2478bc6a349e77", "testCaseId": "ad0130f34bd17b13bbe8d80f846f309c", "fullName": "test_cacu_yaml.TestCalc#test_add", "labels": [{"name": "tag", "value": "run(order=1)"}, {"name": "suite", "value": "test_cacu_yaml"}, {"name": "subSuite", "value": "TestCalc"}, {"name": "host", "value": "DESKTOP-FCPOG81"}, {"name": "thread", "value": "5224-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_cacu_yaml"}]}